core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpEventBuffer.java

```diff
@@ -231,6 +228,27 @@ public long addTimestamp(long address, long value) {
         return address + Long.BYTES + Byte.BYTES;
     }
 
+    public long addUuid(long offset, DirectByteCharSequence value, boolean hasNonAsciiChars) throws NumericException {
+        int maxLen = 2 * value.length();
+        checkCapacity(offset, Byte.BYTES + Integer.BYTES + maxLen);
+        long strPos = offset + Byte.BYTES + Integer.BYTES; // skip field type and string length
+        tempSink.of(strPos, strPos + maxLen);
+        if (hasNonAsciiChars) {
+            utf8ToUtf16Unchecked(value, tempSink);
+        } else {
+            tempSink.put(value);
+        }
+        UuidUtil.checkDashesAndLength(tempSink);
+        long hi = UuidUtil.parseHi(tempSink);
+        long lo = UuidUtil.parseLo(tempSink);
```

Comment:
Numeric exception is unhandled

Reply:
it's handled on the [call-site](https://github.com/questdb/questdb/blob/64454143b0b098a2551b9a1e3afe65f83f1a0d3c/core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java#L467-L473)
that's consistent with the handling of casting errors for other types.